### PR TITLE
Implement ScreenSpot inference pipeline and export results as JSONL

### PR DESCRIPTION
**Pull request by JunseokLee42**
### Summary
This PR adds a complete inference script for the [ScreenSpot dataset](rootsautomation/ScreenSpot).
The script inferences using the `GUI-Actor-7B/2B-Qwen2-VL` and `GUI-Actor-7B/3B-Qwen2.5-VL` and saves the results in `.jsonl` format for downstream evaluation or analysis.

### Changes
- Added a new script `inference-Qwen.py`
- Loads ScreenSpot test set and runs inference using the Qwen model
- Stores instruction, bbox, and predictions as a `.jsonl` file

### Note
- This script currently supports **only the ScreenSpot** dataset.
- To evaluate other datasets, additional implementation is required.

### CODE REVIEW

**Comment by QianhuiWu** [comment]
@JunseokLee42 What's the key difference between this PR and the codes under `eval/screenspot.py`?

**Comment by JunseokLee42** [comment]
@QianhuiWu Oh, you've already implemented the evaluation and inference code. I didn't notice it.
Then, I think the only key difference between them is related to **memory usage** rather than newly implementation on inference.

This PR includes in **`with torch.no_grad()`** function (line 80-81th) to disable gradient computation needed in backward. By using the function, it leads to memory usage reduction in the inference stage. You can compare it with your code **line 120th**.

If the above feature is also implemented in your `eval/screenspot.py`, you can ignore this PR.
